torch/distributed/rpc/api.py

```diff
@@ -66,10 +90,46 @@ def _wait_all_workers():
     terminate the RPC framework, and there is no guarantee that the RPC
     framework will work after this method returns.
     """
-    global _agent
+    if _agent is None:
+        return
```

Comment:
We should throw an exception here instead of silently ignoring this.

Reply:
Adding `@_require_initialized` to it.